src/frontend/web_application/src/components/form/PasswordStrenght/style.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../../styles/common';
+@import '../../../styles/config/config';
+@import '../../../styles/vendor/bootstrap_foundation-sites';
+
+
+$color-weak: map_get($caliopen-palette, 'alert');
+$color-moderate: map_get($caliopen-palette, 'warning');
+$color-strong: map_get($caliopen-palette, 'success');
+
+.m-passw-strenght {
+
+  @include flex-grid-row;
+  margin: 0 0 map-get($co-form__spacing, 'medium');
+
+  &__row {
+    @include flex-grid-column(12);
+  }
+
+  &__text {
+    padding-left: 30px;
+    font-size: $co-font__size--small;
+    font-weight: 700;
+    text-shadow: 0 0 1px $co-color__shadow;
+    text-transform: uppercase;
+  }
+
+  &__graph {
+    @include flex-grid-row;
+    padding-left: 0 !important;
```

Comment:
this might not be here (it looks like a bug mixin row and column and not using `flex-grid-size`)